docs/software/matlabinteractive.md

```diff
@@ -3,58 +3,225 @@ title: MATLAB Interactive
 created: 2023-03-31T09:27:02 (UTC -04:00)
 tags: []
 source: https://sites.wustl.edu/chpc/resources/software/matlab_interactive/
-author:
+author: Benjamin Kay
 exclude: true
 ---
 
-The MATLAB home page is [https://www.mathworks.com/](https://www.mathworks.com/).
+[MATLAB](https://www.mathworks.com/) is a popular programming and numeric computing platform. Matlab is available to cluster users via cluster-wide parallel license.
 
-This is interactive MATLAB that can run directly on either login nodes or compute nodes. **Note that interactive MATLAB is only available to CIL members**.
+> Note: Interactive MATLAB may only allow running serial jobs with one CPU core. If you need to run parallel jobs, use the [MATLAB Parallel Server](https://sites.wustl.edu/chpc/resources/software/matlab_parallel_server/).
```

Comment:
That page no longer exists; please change link to software/matlabparallelserver.md

Reply:
Updated in this and future PRs via force-push